package.json

```diff
@@ -3,7 +3,7 @@
   "description": "The Internet Archive Collection Browser.",
   "license": "AGPL-3.0-only",
   "author": "Internet Archive",
-  "version": "1.14.11",
+  "version": "1.14.11-alpha.10",
```

Comment:
we normally do not commit these alpha tags. as a rule of thumb, alpha version bumps don't really get merged to main...

Reply:
yup... will remove it